.eslintrc

```diff
@@ -1,26 +1,23 @@
 {
-    "root": true,
-    "parser": "@typescript-eslint/parser",
-    "plugins": [
-      "@typescript-eslint",
-      "react",
-      "prettier"
-    ],
-    "extends": [
-      "eslint:recommended",
-      "plugin:@typescript-eslint/eslint-recommended",
-      "plugin:@typescript-eslint/recommended",
-      "plugin:react/recommended",
-      "plugin:react/jsx-runtime",
-      "prettier"
-    ],
-    "rules": {
-        "no-console": 0,
-        "prettier/prettier": 2
-    },
-    "settings": {
-      "react": {
-        "version": "detect"
-      }
+  "root": true,
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["@typescript-eslint", "react", "prettier"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "prettier"
+  ],
+  "rules": {
+    "no-console": 0,
+    "prettier/prettier": 2,
+    "@typescript-eslint/explicit-module-boundary-types": "off"
+  },
+  "settings": {
+    "react": {
+      "version": "detect"
     }
+  }
 }
```

Comment:
Looks like this json was prettified - love it. What do you use to prettify json? Right now the prettier in the repo doesn't touch json files, but I'm on board to add it so we don't end up with different json files with different formatting.

You're adding the `"@typescript-eslint/explicit-module-boundary-types": "off"` rule here, which I'm not opposed to.

I'm also noticing that your touched files use CRLF line endings. Right now there's no preference for what gets committed, but we should probably decide on CRLF or LF line endings so we don't mix and match in the repo. Right now the rest of the files use LF, but it's trivial to change them to CRLF if we choose. We should probably add something to the `.gitattributes` file and `.eslintrc` file based on what we decide. What do you think?
https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings
https://eslint.org/docs/rules/linebreak-style

Reply:
I ran into something with CRLF too. When I initially cloned the repo, prettier told me I had 155 errors all regarding the newline stuff. I edited a git config option with `git config --global core.autocrlf false` and Prettier stopped screaming at me about it

Reply:
I'm using a linter in my editor so whenever I save the file, it gets prettified.

Not sure what CRLF vs LF means but I can change whatever needs to be changed.

Also for the `@typescript-eslint/explicit-module-boundary-types": "off"` I'll change it back. That was an error I ran into before

Reply:
re: CRLF vs LF - I think git will automatically convert LF to CRLF if you checkout files on Windows, but also by default prettier prefers LF line endings. The `core.autocrlf` config refers to git doing this, so setting it to false would retain the LF line endings that are in the `main` files now. I'll go ahead and push a `.gitattributes` file that will enforce `core.autocrlf false` for this repo so contributors don't have to mess with their global git config, and add that as a rule to eslint/prettier so everyone will get the same errors if they try using CRLF endings. This probably won't cause any problems for anyone since text editors are all cross-platform these days ([unless anyone is using Notepad pre-2018](https://devblogs.microsoft.com/commandline/extended-eol-in-notepad/)). If you're using vscode I'd recommend throwing `"files.eol": "\n"` in your `settings.json` if you're ok with setting that default behavior globally.